### ISSUE TITLE
[P3.2][Sourcing][Accessibility][Pretest][Table] The checkbox in the first column has only redundant state information and operating instructions and no labelling (“To deselect row, press SPACEBAR gewählt … Click to deselect” or “To select row, press SPACEBAR … Click to select”).

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10817]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The checkbox in the first column has only redundant state information and operating instructions and no labelling (“To deselect row, press SPACEBAR gewählt … Click to deselect” or “To select row, press SPACEBAR … Click to select”).

The checkbox should be labeled with the content of the second column ("Name") and unnecessary operating instructions and status information should be removed

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
platform multi selection table

**Comment by nikolay-kolarov** [comment]
Hi,

I think the redundancy complaint comes from #10619, but the text for selection needs to be on this cell.
And the focus goes on the cell, not a checkbox, so there is no to relate another column.
At the end, the user is in a table and receives the info that with a space one can activate the whole row.

So if @I543348  agrees this ticket can be closed. 

@droshev - FYI

Best Regards,
Nikolay

**Comment by I543348** [comment]
Hi @nikolay-kolarov ,
As a part of this issue we dont have to keep 2 reading instructions for checkbox 
Currently JAWS is reading "Click to select or deselect" and also " To select a row , Press spacebar " 
I think one should be removed as a feedback that came from customer. (may be "Click to select or deselect" can be removed)
Pls let us know your opinion.
Pls help to reopen if you agree , else we need a reason on what can be communicated to customers.

Thanks.
cc: @sagar7259

**Comment by nikolay-kolarov** [comment]
Hi @I543348,

Basically what I have commented in #10619 will solve this:
"
So one option would be to contsruct a string with the value of the row and pass it to aria-labelledby. This is done on focus in the responsive table in UI5: https://ui5.sap.com/#/entity/sap.m.Table/sample/sap.m.sample.TableMultiSelectMode
The other option is to not allow the cell in the tree on whole row, like it is in the UI5 tree table: https://ui5.sap.com/#/entity/sap.ui.table.TreeTable/sample/sap.ui.table.sample.TreeTable.JSONTreeBinding
"
For the first option, when a custom aria-labelledby is calculated it will not contain the text, so it will be only available on the cell for selsction.
For the second the row will not be focusable, so also the text will be announced only on the cell for selection.
Therefore I think it does not make sense to open this one in addition. You can check the outcome once #10619 is solved.

Best Regards,
Nikolay
